agent/doc/swdesign/README.md

```diff
@@ -2759,6 +2759,22 @@ Needs:
 - impl
 - utest
 
+#### AgentManager sends the node resource availability to the server
+`swdd~agent-sends-node-resource-availability-to-server~1`
+
+Status: approved
+
+At an interval of 2 seconds the AgentManager measures the global cpu load and the available free memory and sends them to the Ankaios server via an `AgentLoadStatus` message.
+
+Rationale: The Ankaios server workloads scheduler shall necessitate in the future the knowledge of available resources.
```

Comment:
We don't intend to add a scheduler to the server itself, but enable writing a scheduler as a workload:
```suggestion
Rationale:
Available resources must be available in the cluster in order to enable dynamic scheduling, e.g., done by a workload.
```